sdcm/utils/docker_utils.py

```diff
@@ -384,6 +384,20 @@ def get_container_name_by_id(cls, c_id: str, docker_client: DockerClient = None)
         docker_client = docker_client or cls.default_docker_client
         return docker_client.containers.get(c_id).name
 
+    @classmethod
+    def pause_container(cls, instance: object, name: str):
+        if name not in instance._containers:
+            raise NotFound(f"{instance}: there is no container registered for name `{name}'")
+        container = cls.get_container(instance, name)
+        container.pause()
```

Comment:
Don't need to do the check, because it's done in `cls.get_container()`. So, it can be one-liner:
`cls.get_container(instance, name).pause()`

Reply:
fixed